docs-www/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@brainhubeu/gatsby-docs-kit": "3.0.8",
     "@brainhubeu/react-carousel": "^1.19.14",
-    "gatsby": "2.20.24",
+    "gatsby": "^2.23.20",
```

Comment:
sometimes there were problems with `gatsby` so we decided to use an exact version

Reply:
In the past they were issues with gatsby versions 2.20.24+ but the current version seems to work